STUDIO/org.openl.rules.webstudio/webapp/pages/modules/table.xhtml

```diff
@@ -486,7 +486,7 @@
                 <td>
                     <h:form>
                         <a4j:commandLink action="#{tableCreatorWizardManager.startWizard}"
-                            styleClass="toolbarButton" title="Create a new Test table"
+                            styleClass="toolbarButton menuButton" title="Create a new Test table"

```

Comment:
Why we need it here? menuButton class is used in buttons with dropdown menu (Run/Trace/Benchmark in Test table).
"Create Test Table" button doesn't have such menu.

Reply:
Mistatke